packages/webview_flutter/webview_flutter_wkwebview/test/src/web_kit/web_kit_test.dart

```diff
@@ -96,6 +107,69 @@ void main() {
       });
     });
 
+    group('$WKHttpCookieStore', () {
```

Comment:
Do you know where this convention of using the `$` to make this an actual class reference comes from? I found in the past that it really messed up VS Code's test UI, and I don't see any advantage, so unless there's some reason to do it I would rather we not do it going forward, and just name the groups with regular strings.

(I missed this in previous reviews.)

Reply:
I believe I started this so the test name would change when the class name changed. But, I fine with changing the practice. I updated all the tests.